modules/nw-exposing-router-metrics.adoc

```diff
@@ -0,0 +1,157 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress_operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-exposing-router-metrics_{context}"]
+= Exposing router metrics
+
+You can expose the HAProxy router metrics by default in Prometheus format on the default stats port, 1936. The external metrics collection and aggregation systems such as Prometheus and statsd can access the HAProxy router metrics. You can view the HAProxy router metrics in a browser in the HTML and cluster service version (CSV) format.
```

Comment:
Is statsd still relevant in 4.x or was that 3.x specific? I ask because I don't see another reference to statsd currently in the docs repo (at least in more recent 4.x branches).